src/garage/torch/policies/gaussian_mlp_policy.py

```diff
@@ -88,58 +89,70 @@ def __init__(self,
                                    layer_normalization=layer_normalization)
 
     def get_action(self, observation):
-        """Get a single action given an observation.
+        r"""Get a single action given an observation.
 
         Args:
             observation (np.ndarray): Observation from the environment.
+                Shape is :math:`env_spec.observation_space`.
 
         Returns:
             tuple:
-                * np.ndarray: Predicted action.
+                * np.ndarray: Predicted action. Shape is
+                    :math:`env_spec.action_space`.
                 * dict:
-                    * list[float]: Mean of the distribution
-                    * list[float]: Standard deviation of logarithmic values of
-                        the distribution
+                    * np.ndarray[float]: Mean of the distribution
+                    * np.ndarray[float]: Standard deviation of logarithmic
+                        values of the distribution.
 
         """
         with torch.no_grad():
+            if not isinstance(observation, torch.Tensor):
+                observation = torch.as_tensor(observation).float().to(
+                    tu.global_device())
             observation = torch.Tensor(observation).unsqueeze(0)
             dist = self.forward(observation)
             return (dist.rsample().squeeze(0).numpy(),
                     dict(mean=dist.mean.squeeze(0).numpy(),
                          log_std=(dist.variance**.5).log().squeeze(0).numpy()))
 
     def get_actions(self, observations):
-        """Get actions given observations.
+        r"""Get actions given observations.
 
         Args:
             observations (np.ndarray): Observations from the environment.
+                Shape is :math:`batch_dim \bullet env_spec.observation_space`.
 
         Returns:
             tuple:
                 * np.ndarray: Predicted actions.
+                    :math:`batch_dim \bullet env_spec.action_space`.
                 * dict:
-                    * list[float]: Mean of the distribution
-                    * list[float]: Standard deviation of logarithmic values of
-                        the distribution
+                    * np.ndarray[float]: Mean of the distribution.
+                    * np.ndarray[float]: Standard deviation of logarithmic
```

Comment:
uhh this sounds wrong. it's it the log of the std? you can also use equations instead.